src/api_proxy/service_control/request_builder.cc

```diff
@@ -1383,10 +1383,6 @@ Status RequestBuilder::ConvertCheckResponse(
     case CheckError::RESOURCE_EXHAUSTED:
       check_response_info->error_type = ScResponseErrorType::CONSUMER_QUOTA;
       return Status(Code::RESOURCE_EXHAUSTED, "Quota check failed.");
-    case CheckError::ABUSER_DETECTED:
```

Comment:
Just double checking, there were no new codes added in the proto?

Reply:
Checked, nop.

https://github.com/googleapis/googleapis/commit/1db88f2a42f6b0911c26eaf83e77377252cf94cc